### PR TITLE
Added login for assessment takers.

### PR DESCRIPTION
**Pull request by ppuarat**

### CODE REVIEW

**Comment by ppuarat** [comment]
image

File: src/views/dashboardAccountablePersonSubPage/Assessment.jsx [hunk 1]
```diff
@@ -28,35 +29,38 @@ class Assessment extends React.Component {
...
     this.api = new Api();
-    document.documentElement.classList.remove("nav-open");
   }
 
   componentDidMount() {
     this.subscribedPolicyId = this.props.match.params.subscribedPolicyId;
     this.api.fetchSubscribedPolicy(this.subscribedPolicyId)
       .then(response => {
-        console.log(response)
+        console.log(response.data);
```
Comment (AntonRozhanskii) [inline src/views/dashboardAccountablePersonSubPage/Assessment.jsx]:
```suggestion
```

File: src/views/dashboardAccountablePersonSubPage/Assessment.jsx [hunk 2]
```diff
@@ -83,45 +87,44 @@ class Assessment extends React.Component {
       })
   }
 
+
```
Comment (AntonRozhanskii) [inline src/views/dashboardAccountablePersonSubPage/Assessment.jsx]:
```suggestion
```

File: src/views/dashboardUserSubPage/SubscribedPolicies.jsx [hunk 3]
```diff
@@ -72,13 +78,16 @@ class SubscribedPolicies extends React.Component {
...
+              {
+                policy.status === "awareness" &&
+                <Button className="btn-round"
+                  style={{ 'marginRight': '7px' }}
+                  color="success"
+                  value={policy.policy_name}
+                  onClick={(e) => this.assessmentButtonHandler(e, policy._id)}>
+                  Send Assessment
+              </Button>
+              }
```
Comment (ppuarat) [inline src/views/dashboardUserSubPage/SubscribedPolicies.jsx:90]: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

File: src/views/dashboardAccountablePersonSubPage/Assessment.jsx [hunk 4]
```diff
@@ -28,35 +29,38 @@ class Assessment extends React.Component {
...
           assessment: response.data.assessments
         });
       }).catch(function (error) {
         console.log(error);
       });
+
   }
   componentDidUpdate() {
-    document.body.classList.remove("register-page");
   }
```
Comment (AntonRozhanskii) [inline src/views/dashboardAccountablePersonSubPage/Assessment.jsx]:
```suggestion
```

File: src/views/dashboardAccountablePersonSubPage/Assessment.jsx [hunk 5]
```diff
@@ -28,35 +29,38 @@ class Assessment extends React.Component {
...
         this.setState({
           currentPolicy: response.data,
           assessment: response.data.assessments
         });
       }).catch(function (error) {
         console.log(error);
       });
+
   }
   componentDidUpdate() {
```
Comment (AntonRozhanskii) [inline src/views/dashboardAccountablePersonSubPage/Assessment.jsx]:
```suggestion
```

File: src/views/dashboardAccountablePersonSubPage/Assessment.jsx [hunk 6]
```diff
@@ -83,45 +87,44 @@ class Assessment extends React.Component {
...
+      if (checkedAnswer == correct_answer) {
+        option.isSelected = true;
+      } else {
+        option.isSelected = false;
       }
-      else {
-        isAnswerCorrect.push(false);
-      }
-      console.log("ASDF" + isAnswerCorrect)
+      console.log(option);
```
Comment (AntonRozhanskii) [inline src/views/dashboardAccountablePersonSubPage/Assessment.jsx]:
```suggestion
```

**Review by AntonRozhanskii — CHANGES_REQUESTED** [review]

**Review by AntonRozhanskii — APPROVED** [review]

**Pull request merged**